network/bridge_endpointclient_linux.go

```diff
@@ -174,3 +177,33 @@ func (client *LinuxBridgeEndpointClient) DeleteEndpoints(ep *endpoint) error {
 
 	return nil
 }
+
+func AddRuleToRouteViaHost(epInfo *EndpointInfo) error {
```

Comment:
AddRuleToRouteViaHost- make this to addRuleToRouteViaHost
The reason is in golang, if function/variable start with caps, its like exported function/variable.